default.nix

```diff
@@ -40,6 +40,7 @@
 , dockerTag ? null
 }:
 rec {
+  inherit cargo sources pkgs;
```

Comment:
@nightkr just realised `default.nix` comes from [operator-templating](https://github.com/stackabletech/operator-templating/blob/main/template/default.nix)

Should I revert this change, and pull in pkgs from sources?

```suggestion
```

Reply:
Oh no, because shell.nix needs at least `cargo` from `default.nix`.
So should this line go into operator-templating?

Reply:
Done in https://github.com/stackabletech/operator-templating/pull/323